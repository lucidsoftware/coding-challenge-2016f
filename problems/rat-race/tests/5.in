3
7 7
#######
#####.#
#####.#
#####.#
#####.#
#####.#
#######
10 10
##########
##########
##########
##########
##########
##########
#######..#
#######.##
#####....#
##########
12 12
############
############
###.##.#####
###.##.#####
###....#####
###.##.#####
###.##.#####
######...###
############
############
############
############