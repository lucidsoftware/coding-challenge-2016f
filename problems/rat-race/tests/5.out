Maximum path length is 4
Maximum path length is 5
Maximum path length is 10
